ivadomed/loader/utils.py

```diff
@@ -365,12 +369,20 @@ def _get_label(dataset, idx):
         Returns:
             int: 0 or 1.
         """
-        # For now, only supported with single label
-        sample_gt = np.array(dataset[idx]['gt'][0])
-        if np.any(sample_gt):
-            return 1
+        if metadata != 'gt':
```

Comment:
What about adding above this: two lines that check if `metadata` is either `gt` either in `dataset[idx]['input_metadata'][0].keys()`, otherwise throw an error?

Reply:
> Smooth and easy! Many thanks!
> Would you simply mind adding a line about this new feature in the RTD/config_file.. and then ready to go :-)

Updated here: c43e47b

Reply:
I added a metadata check in the loader/loader.py : 9096673. Doing the check directly in the BalancedSampler was useless since if the metadata is not 'gt' or a column from participants.tsv an error was already thrown in loader/loader.py, so I added a custom error :}